app/javascript/controllers/step_sortable_controller.js

```diff
@@ -0,0 +1,21 @@
+import { Controller } from "@hotwired/stimulus"
+import Sortable from "sortablejs"
+
+export default class StepSortable extends Controller {
+  static targets = ["container"]
+
+  connect() {
+    this.sortable = Sortable.create(this.containerTarget, {
+      animation: 150,
+      onEnd: this.reorder.bind(this),
+      handle: ".step-position-handle"
```

Comment:
```suggestion
      handle: ".ui-sortable-handle"
```

histoire de rester iso avec le reste